uwblueprint/community-fridge-kw
# fix: Update style of buttons to match Figma

## Brief description. What is this change? 

### [Edit Scheduling Screen does not match designs](https://www.notion.so/uwblueprintexecs/UI-Edit-Scheduling-Screen-does-not-match-designs-e12d57e68bcb47f79e5edd111c29ab9a)



## Implementation description. How did you make this change?
* Moved position of save / next buttons
* Made all buttons styling consistent and adhering to Figma
* Removed progress bar from the pages when you are editing a donation



## Steps to test
1. Schedule a donation: Check that all back and next buttons are working as expected
3. Editing donation: Check that all cancel and save buttons are working



## Checklist
